dora/src/vm/annotations.rs

```diff
@@ -42,7 +42,8 @@ pub struct Annotation {
     pub name: Name,
     pub namespace_id: NamespaceId,
     pub ty: SourceType,
-    pub internal: Option<Modifier>,
+    pub internal: bool,
```

Comment:
Do we need both fields? Shouldn't `internal == internal_annotation.is_some()` hold?

Reply:
I think it was intended to reduce confusion between an internal annotation and *the* internal annotation.

I removed it.